### DISCUSSION TITLE
osu!lazer Mouse+Tablet left click won't hold down

### DISCUSSION THREAD

**Discussion by CesiumCs** [ppy/osu#24318] [Q&A]
I'm left handed and it's easiest for me to play with my left hand holding a pen, and my right hand holding the mouse. At some point, an issue came up where when my pen is hovering over my tablet, left mouse will only register a very short click, and I stopped playing for a bit thinking it was a bug and it would be fixed. A year or so later, I downloaded lazer and tried to play again, and am still running into the issue. Using 2023.717.0-lazer on Arch Linux with opentabletdriver installed on my system. Right click works as normal - I can click and drag using it.

**Comment by peppy** [comment]
Are you using the built-in tablet support?

If not, please do.
